site/components/Search.jsx

```diff
@@ -88,7 +88,7 @@ class Search extends React.Component {
         let classes = cx({
             'D(ib) Mstart(3px) Ov(h) Va(m) Pos(a) End(20px)': true,
             'W(0)': this.state.visible === false,
-            'W(200px) Ov(a)': this.state.visible
+            'W(200px) Ov(h)': this.state.visible
```

Comment:
Looks like we can just remove the `Ov(h)` here since its already specified in the `true` case.